### PR TITLE
Test Secure login go

### PR DESCRIPTION
**Pull request by Nstsy**
-add test to validate login form with empty data
-add test to validate login form with any email
-add test to validate login form with any password
-add test to validate login form with any data

### CODE REVIEW

File: src/test/java/gov/login/secure/LoginTest.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    @Test
+    public void test1() {
+        WebDriver webDriver = new ChromeDriver();
+        webDriver.get("https://secure.login.gov/");
+        String buttonXpath = "//*[@id=\"new_user\"]/lg-submit-button/button";
+        By buttonBy = By.xpath(buttonXpath);
+        WebElement buttonWebElement = webDriver.findElement(buttonBy);
+        buttonWebElement.click();
+    }
+    @Test
```
Comment (github4ta) [inline src/test/java/gov/login/secure/LoginTest.java:19]: методы разделяются пустой строкой

File: src/test/java/gov/login/secure/LoginTest.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+    @Test
+    public void test3() {
+        WebDriver webDriver = new ChromeDriver();
+        webDriver.get("https://secure.login.gov/");
+        String inputPasswordXpath = "/html/body/main/div/form/lg-password-toggle/lg-validated-field/div/input";
+        By inputPasswordBy = By.xpath(inputPasswordXpath);
+        WebElement inputPasswordWebElement = webDriver.findElement(inputPasswordBy);
+        inputPasswordWebElement.sendKeys("1q2w#E4r");
+        String buttonXpath = "//*[@id=\"new_user\"]/lg-submit-button/button";
```
Comment (github4ta) [inline src/test/java/gov/login/secure/LoginTest.java:41]: отдельное описание для разных веб елементов можно разделить пустой строкой для удобства чтения кода
